vms/platformvm/utxo/camino_locked.go

```diff
@@ -38,7 +38,7 @@ var (
 	errInputsUTXOsMismatch       = errors.New("number of inputs is different from number of utxos")
 	errBadCredentials            = errors.New("bad credentials")
 	errNotBurnedEnough           = errors.New("burned less tokens, than needed to")
-	errAssetIDMismatch           = errors.New("utxo/input/output assetID is different from expected asset id")
+	errUnexpectedAssetID         = errors.New("utxo/input/output assetID is different from expected asset id")
```

Comment:
Avalanche cortina 1 introduces new error var with exactly the same name, but with different description and meaning. This var is renamed in order to keep them segregated.